docs/reference/airnode/latest/developers/verify-airnode-addresses.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Verify Airnode addresses
+sidebarHeader: Reference
+sidebarSubHeader: Airnode
+pageHeader: Reference → Airnode → v0.11 → Verify Airnode Addresses
+path: /reference/airnode/latest/developers/verify-airnode-addresses.html
+version: v0.11
+outline: deep
+tags:
+---
+
+<VersionWarning/>
+
+<PageHeader/>
+
+<SearchHighlight/>
+
+<FlexStartTag/>
+
+# {{$frontmatter.title}}
+
+An Airnode is owned and operated by an API provider. Every Airnode has a unique
+on-chain
+[`airnodeAddress`](/reference/airnode/latest/concepts/airnode.html#airnodeaddress)
+created and known by its API provider. API providers normally announce their
+Airnode addresses in the form of a DNS `TXT` record within their DNS
+registration thus proving ownership. An Airnode's address is the same for all
+chains.
+
+The key for the `TXT` record is `chainapi-verification-keys` and is an array of
+Airnode addresses owned by the API provider. Below is an example of the Airnode
+for
+[dxFeed<ExternalLinkImage/>](https://www.digitalocean.com/community/tools/dns?domain=dxfeed.com#TXT-Records).
+
+![dxfeed-txt-record](../assets/images/txt-record-dxfeed.png)
```

Comment:
I suggest cropping this image to just the relevant line, otherwise there is a lot of irrelevant data.

Reply:
Done